backend/sirius-web-collaborative-diagrams-api/src/main/java/org/eclipse/sirius/web/collaborative/diagrams/api/dto/DeleteFromDiagramInput.java

```diff
@@ -22,17 +23,20 @@
 import org.eclipse.sirius.web.collaborative.diagrams.api.IDiagramInput;
 
 /**
- * The input for the "Delete from diagram" mutation.
+ * The class of the inputs for the "Delete from diagram" mutation.
  *
  * @author pcdavid
+ * @author hmarchadour
  */
 @GraphQLInputObjectType
 public final class DeleteFromDiagramInput implements IDiagramInput {
     private UUID projectId;
 
     private UUID representationId;
 
-    private String diagramElementId;
+    private List<String> nodeIds;
+
+    private List<String> edgeIds;
```

Comment:
Why not List of UUID?

Reply:
This implies a GraphQLID update that I've not think todo.

Reply:
@see following conversation